src/smartcontracts/smartContract.local.net.spec.ts

```diff
@@ -325,4 +330,51 @@ describe("test on local testnet", function () {
         queryResponse = await provider.queryContract(query);
         assert.equal(decodeUnsignedNumber(queryResponse.getReturnDataParts()[0]), 0);
     });
+
+    it("counter: should deploy and call using the SmartContractFactory", async function () {
+        this.timeout(80000);
+
+        TransactionWatcher.DefaultPollingInterval = 5000;
+        TransactionWatcher.DefaultTimeout = 50000;
+
+        const network = await provider.getNetworkConfig();
+        await alice.sync(provider);
+
+        const transactionComputer = new TransactionComputer();
+        const config = new TransactionsFactoryConfig(network.ChainID);
```

Comment:
Now I see - we should pass init parameters to `TransactionsFactoryConfig` as options (even if now there's only one, the chain ID) - similar to `SmartContractTransactionsFactory` etc. This, in order to avoid breaking changes in the future, if the config should be extended to receive an extra parameter in the constructor.

This would be for another, separate PR.